packages/wrangler/src/index.ts

```diff
@@ -1483,6 +1483,13 @@ export function createCLIParser(argv: string[]) {
 
 	wrangler.exitProcess(false);
 
+	Object.defineProperty(wrangler, "_registry", {
+		value: registry,
+		enumerable: false,
+		writable: false,
+		configurable: false,
+	});
+
 	return wrangler;
```

Comment:
Don't do this. Instead, change the signature of `createCLIParser()` to return `{ wrangler, registry }`